### PR TITLE
Include option for padAngle on RadialChart

### PR DESCRIPTION
**Pull request by dvnrsn**
Currently there are no means to add padding between arcs on a RadialChart. This PR seeks to enable such capability.

### CODE REVIEW

File: src/plot/series/arc-series.js [hunk 1]
```diff
@@ -180,6 +182,7 @@ class ArcSeries extends AbstractSeries {
             endAngle: angle(row)
           };
           const arcedData = arcBuilder();
```
Comment (mcnuttandrew) [inline src/plot/series/arc-series.js]: nit: maybe make this a one liner arcBuilder().padAngle(padAngle)

File: src/radial-chart/index.js [hunk 2]
```diff
@@ -191,6 +191,7 @@ RadialChart.propTypes = {
   ).isRequired,
   getAngle: PropTypes.func,
   getAngle0: PropTypes.func,
+  padAngle: PropTypes.number,
```
Comment (mcnuttandrew) [inline src/radial-chart/index.js]: Maybe set the default padAngle on here too?

**Comment by mcnuttandrew** [comment]
It looks like the CLA is mad at you. I think you can get around this by squashing this whole branch into a single commit

**Comment by dvnrsn** [comment]
Update email but can't get CLA to play nicely. Going to refork and create new PR...

**Comment by dvnrsn** [comment]
`padAngle={0}`
screen shot 2018-08-21 at 10 50 14
`padAngle={0.03}`
screen shot 2018-08-21 at 10 50 49

**Comment by dvnrsn** [comment]
Is [showcase code](https://github.com/uber/react-vis/blob/master/showcase/radial-chart/custom-radius-radial-chart.js) rendered somewhere? Is this the place where I should add padAngle? @mcnuttandrew

File: docs/radial-chart.md [hunk 3]
```diff
@@ -67,6 +67,10 @@ SVG paths (which is what the arc series is made up of) have numerous manipulable
 Type: `string`
 The className to be added to an individual arc in the series.
 
+#### padAngle (optional)
+Type: `number|function`
+The padding to be applied between arcs.
```
Comment (mcnuttandrew) [inline docs/radial-chart.md]: I think including some text here like, "see above donut chart for an example of a padded angle" would be fine

**Comment by mcnuttandrew** [comment]
Alright looks good! Stamp stamp stamp!!!!

**Pull request merged**